Tests/CarthageKitTests/DependencySpec.swift

```diff
@@ -213,12 +213,14 @@ class DependencySpec: QuickSpec {
 					expect(dependency) == .binary(URL(string: "file:///my/domain/com/framework.json")!)
 				}
 
-				it("should fail with non-https URL") {
-					let scanner = Scanner(string: "binary \"nope\"")
+				it("should read a URL with relativeFile scheme") {
+					let scanner = Scanner(string: "binary \"my/domain/com/framework.json\"")
 
-					let error = Dependency.from(scanner).error
+					let dependency = Dependency.from(scanner).value
```

Comment:
@blender what are you thoughts on injecting current dir to provide dependency injection of CWD for these relative paths? This way I wouldn't have to insert duplicated URL construction logic between test vs Dependency being responsible?